packages/ihe-gateway/server/CodeTemplates/XCA Library - Inbound/Get XCA ITI-39 RetrieveDocumentSetResponse/Get XCA ITI-39 RetrieveDocumentSetResponse.js

```diff
@@ -7,48 +7,59 @@
 	@return {Object} return XCA ITI-39 response payload
 */
 function getXCAITI39QueryResponse(request, operationOutcome, mtom) {
+    var _response = <ihe:RetrieveDocumentSetResponse xmlns:ihe="urn:ihe:iti:xds-b:2007">
+                <RegistryResponse xmlns="urn:oasis:names:tc:ebxml-regrep:xsd:rs:3.0" status="urn:oasis:names:tc:ebxml-regrep:ResponseStatusType:Success"/>
+            </ihe:RetrieveDocumentSetResponse>;
 
-	var _response = <ihe:RetrieveDocumentSetResponse xmlns:ihe="urn:ihe:iti:xds-b:2007">
-					<RegistryResponse xmlns="urn:oasis:names:tc:ebxml-regrep:xsd:rs:3.0" status="urn:oasis:names:tc:ebxml-regrep:ResponseStatusType:Success"/>
-				</ihe:RetrieveDocumentSetResponse>;
+    if (mtom) {
```

Comment:
if mtom lets ignore it for now since we don't really know how to handle that or what it is well enough. Never actually implemented it properly